newbie_guide.md

```diff
@@ -0,0 +1,212 @@
+    ```
+
+    Verify that balance is now non-zero:
+
+    ```sh
+    bitcoin-cli -regtest getbalance
+    ```
+
+    If balance is still zero, generate 100 more blocks.
+
+## Step 2. Compile & initial run
+
+- Be sure to have [`exonum_launcher`](https://github.com/popzxc/exonum-launcher)
+installed and updated to the point at which it supports plugins.
+- Install `exonum_btc_anchoring_plugin`
+    (if you're using `venv`, activate `venv` in which `exonum_launcher` is installed):
+
+    ```sh
+    pip install -e exonum-btc-anchoring/launcher
+    ```
+
+- Compile and install `btc_anchoring` example:
+
+    ```sh
+    cd exonum-btc-anchoring
+    cargo install --path . --example btc_anchoring --force
+    ```
+
+- Build the `exonum-btc-anchoring` project (it contains a binary that will be used later):
+
+    ```sh
+    cargo build
+    ```
+
+- Run the example
+
+    ```sh
+    RUST_LOG="exonum_btc_anchoring=info" btc_anchoring run-dev -a anchoring
+    ```
+
+    `-a anchoring` here denotes the directory in which node data will be generated.
+
+## Step 3. Deploying & running
+
+- First of all, we need to obtain `service_key` and `bitcoin_key`.
+
+    To obtain `service_key`, go to the directory in which node data lies
+    (in our example it's `anchoring`) and open `node.toml`.
+    Here you can find `service_key`.
+
+    To obtain `bitcoin_key`, go to the `exonum-btc-anchoring` directory and launch
+
+    ```sh
+    cargo run -- generate-config -o anchoring/sync.toml --bitcoin-rpc-host http://localhost --bitcoin-rpc-user user --bitcoin-rpc-password password
+    ```
+
+    Replace `anchoring` here with the directory in which node data lies.
+
+    As a result of this call you will obtain `bitcoin_key`.
+- Create file `anchoring.yml` with the following contents:
+
+    ```yaml
+    plugins:
+      runtime: {}
+      artifact:
+        anchoring: "exonum_btc_anchoring_plugin.AnchoringInstanceSpecLoader"
+
+    networks:
+      - host: "127.0.0.1"
+        ssl: false
+        public-api-port: 8080
+        private-api-port: 8081
+
+    deadline_height: 10000
+
+    artifacts:
+      anchoring:
+        runtime: rust
+        name: "exonum-btc-anchoring:0.12.0"
+
+    instances:
+      anchoring:
+        artifact: anchoring
+        config:
+          network: regtest
+          anchoring_interval: 500
+          transaction_fee: 10
+          anchoring_keys:
+            - bitcoin_key: "0332ab15173cf9ff8ad0946fbd515434bf1f04bb46482453474b6c38b94fa9d7b7"
+              service_key: "2b89c8e1f3a6a8f18ac3276b87403e39c54c33d8275e9626ab9478df4d6d5bfc"
+    ```
+
+    Replace `bitcoin_key` and `service_key` with values obtained in the previous step.
+- Run the `exonum_launcher` to start & deploy instance:
+
+    ```sh
+    python3 -m exonum_launcher -i anchoring.yml
+    ```
+
+    If everything was done correctly, service should start successfully.
+
+    However, it was only launched, and anchoring does not working yet.
```

Comment:
Present Indefinite with present continuous ? Maybe `...does not work yet.` or `... is not working yet.` ?

Reply:
Good catch